### DISCUSSION TITLE
how to get data context from converter?

### DISCUSSION THREAD

**Discussion by lanyusan** [AvaloniaUI/Avalonia#9467] [Q&A] [answered]
I have a case where the convertion result would rely on some state in data context. Is there a way to get it?

I have try to pass it as converter parameter but according to issue https://github.com/AvaloniaUI/Avalonia/issues/6840 it is not possible now.

Is there a way to get current control's data context from converter?

Thanks

**Comment by kekekeks** [comment, marked as answer]
You need to use MultiBinding and IMultiValueConverter

**Reply by lanyusan** [reply]
Thanks. Looks it is  the only way.
